float64 x_center
float64 y_center
float64 z_center
float64 roll_center
float64 pitch_center
float64 yaw_center
float64 start_angle
float64 end_angle
float64 radius
float64 vel
float64 accl
string profile